### DISCUSSION TITLE
UniFi OS 2.4.27

### DISCUSSION THREAD

**Discussion by rsockanc** [SierraSoftworks/tailscale-udm#52] [General]
I upgraded my UDM Pro to UniFi OS 2.4.27 and then installed tailscale-udm and it is working properly for other endpoints as an exit node. I have several other USG3s with tailscale installed and functioning with advertised routes that I plan to set up in a site-to-site configuration however it seems I am not able to set up subnet routing of local subnets to the other USGs as lightly mentioned in the FAQs that were written prior to the release of 2.4.27 on a UDM Pro.  Perhaps @tomvoss  has tested 2.4.27?  The current issue is that the UDM shows tailscale status as up and it sees its peers yet cannot ping the tailscale peer IPs.

**Comment by tomvoss** [comment]
I'm using [unifios-tailscale](https://github.com/gridironsolutions/unifios-tailscale). I found an issue during install that I think was caused by not having a proper .env file setup before running the install script. Once that was fixed the install succeeded.

The first 2.4.27 router I ran this on experienced recurring, seemingly random issues where Tailscale routes suddenly stopped working. Ultimately I reverted that router to older firmware and an older release of unifios-tailscale. I still don't know if the issue was directly related to unifios-tailscale or not.

I've since upgraded two other routers to 2.4.27 and installed unifios-tailscale and they are working perfectly. So I'll give it a YMMV rating. I'm monitoring the situation before I upgrade the rest of my routers.
